06_gpu_and_ml/reinforcement-learning/grpo-verl.py

```diff
@@ -0,0 +1,297 @@
+# ---
+# cmd: ["modal", "run", "06_gpu_and_ml/reinforcement-learning/grpo-verl.py::train"]
+# ---
+
+# # Run GRPO on Modal using VERL
+
+# This example demonstrates how to run [GRPO](https://arxiv.org/pdf/2402.03300) on Modal using the [VERL](https://github.com/volcengine/verl) framework.
```

Comment:
"run GRPO" should be sth like "train with GRPO" or "run GRPO training"? Otherwise it's incomplete.